modules/promo-code/promo-code.model.js

```diff
@@ -9,6 +9,7 @@ const promoCodeSchema = new mongoose.Schema({
   dateTo: String,
   code: String,
   discount: Number,
+  categories: [String],
```

Comment:
what will happen if somebody puts here value "blablabla"?

Reply:
also, add this field to the category validator

Reply:
it will be done with another issue